packages/Collapsible/src/Collapsible.styles.js

```diff
@@ -0,0 +1,48 @@
+import tokens from "@paprika/tokens";
+import stylers from "@paprika/stylers";
+
+const collapsibleStyles = props => `
+  box-sizing: border-box;
+  color: ${tokens.color.black};
+  display: block;
+  ${stylers.fontSize()};
+  line-height: 24px;
+  width: 100%;
+
+  > .raw-button {
```

Comment:
I don't think the `RawButton` has this class... but I think `.collapsible__label` will suit your purpose instead.